usr/share/rear/prep/BAREOS/default/500_check_bareos_client_configured.sh

```diff
@@ -0,0 +1,32 @@
+#
+# Check that Bareos is installed and configured
+#
+
+mapfile -t clients < <( bcommand ".clients" )
+
+if (( ${#clients[@]} == 0 )); then
+    Error "No Bareos clients found"
+fi
+
+if [ "$BAREOS_CLIENT" ]; then
+    LogPrint "Using $BAREOS_CLIENT as BAREOS_CLIENT"
```

Comment:
Maybe move this down to the end of this file to indicate the result and also cover the automatic configuration? I'm wondering why only this case has a `LogPrint` statement and the others only report on errors.

I think stating the fact that the code used a configuration variable doesn't really tell the user much.

Reply:
Done.